app/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/capture_chrn_page.scala.html

```diff
@@ -45,24 +47,31 @@
 
         @h1(messages("chrn.title"))
 
-        @inset(messages("chrn.inset"))
+        @p(Html(messages("chrn.p1", "<https://www.gov.uk/find-charity-information>")))
+
+        @p(Html(messages("chrn.p2")))
+
+        @p(link(
+                trustControllersRoutes.CaptureCHRNController.noChrn(journeyId).url,
+                messages("chrn.no_chrn_link"),
+                Some("no-chrn")
+            ),
+            classes = "govuk-inset-text"
+        )
 
         @govukInput(
             Input(
-                hint = Some(Hint(content = HtmlContent(s"""<p class="govuk-body">${messages("chrn.hint")}</p>"""))),
-                label = Label(content = Text(messages("chrn.label")), isPageHeading = false, classes = "govuk-label govuk-!-font-weight-bold"),
+                hint = Some(Hint(content = HtmlContent(messages("chrn.hint")))),
+                label = Label(content = Text(messages("chrn.label")), isPageHeading = false, classes = "govuk-fieldset__legend--m"),
                 classes = "govuk-input--width-20"
```

Comment:
The width looks smaller on the prototype. Can you check what the designers want?